extension-localization/src/main/java/com/mapbox/maps/extension/localization/SupportedLanguages.kt

```diff
@@ -136,5 +143,9 @@ internal fun getLanguageNameV8(locale: Locale): String {
     return if (locale == Locale.TAIWAN || locale.script == "Hant") NAME_ZH_HANT
     else NAME_ZH_HANS
   }
-  "name_${locale.language}".apply { return if (supportedV8.contains(this)) this else NAME }
+  return "name_${locale.language}".apply {
+    if (!supportedV8.contains(this)) {
+      Logger.w(TAG, "Language ${locale.displayLanguage} is not supported in the current style")
```

Comment:
would it be enough to log a warning message here?

Reply:
Any suggestions for this?

Reply:
From my perspective, a warning message is enough because users can then switch to other languages.